tests/clickhouse-test

```diff
@@ -377,6 +378,76 @@ def check_server_started(client, retry_count):
     return False
 
 
+class BuildFlags(object):
+    THREAD = 'thread-sanitizer'
+    ADDRESS = 'address-sanitizer'
+    UNDEFINED = 'ub-sanitizer'
+    MEMORY = 'memory-sanitizer'
+    DEBUG = 'debug-build'
+    UNBUNDLED = 'unbundled-build'
```

Comment:
BTW, maybe it's also possible to add ARCADIA.

Reply:
Otherwise we will have separate skip list in different format for ARCADIA.
(we can do it in another PR)

Reply:
let's do it in another PR.